stl/sheldon_team/sheldon/src/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
ref: https://google.github.io/styleguide/cppguide.html#Namespaces

You may not use a using-directive to make all names from a namespace available.

Reply:
Sorry. I have modified this place according to the ref in the below commit. Please check it.